apps/studio/src/features/editing-experience/components/TipTapComponent.tsx

```diff
@@ -174,7 +168,13 @@ function TipTapComponent({
         w="100%"
         justifyContent="end"
       >
-        <Button onClick={() => onProceed(path, editor.getJSON().content)}>
+        <Button
+          onClick={() => {
+            // setPageState()
+            console.log('saving')
+            setDrawerState({ state: 'root' })
+          }}
```

Comment:
slight note: the behaviour changed here. previously, we `onProceed(path)` but now we set the drawer state instead?

Reply:
Yeah the previous onProceed was this (console.log + return to root drawer), the only thing is that we don't log the path anymore